pkgs/test_core/lib/src/runner/engine.dart

```diff
@@ -293,6 +302,30 @@ class Engine {
     return success;
   }
 
+  Future<Null> _gatherCoverage(LiveSuiteController controller) async {
+    if (_coverage == null) return;
+
+    final RunnerSuite suite = controller.liveSuite.suite;
+
+    if (!suite.platform.runtime.isDartVM) return;
+
+    // for some reason the observatoryUrl URI doesn't have the query parameters populated
+    final String isolateId =
+        Uri.parse(suite.environment.observatoryUrl.fragment)
+            .queryParameters['isolateId'];
+
+    final Map<String, dynamic> cov = await collect(
```

Comment:
`collect()` is already typed as `Future<Map<String, dynamic>>` - can we omit the type on the LHS here?

Reply:
Fixed